core/modules/system/js/modules.js

```diff
@@ -73,6 +73,9 @@ Backdrop.behaviors.moduleFilterByText = {
       $rows = $form.find('tbody tr');
       $fieldset = $form.find('fieldset');
 
+      // On page load filter if query string is present.
+      $input.focus().on('select', filterModuleList);
```

Comment:
This doesn't actually affect on page load. We can instead just manually trigger a keyup event on the field:

```
$input.triggerHandler('keyup');
```

Right after binding the event below.